format/src/main/scala/org/scalafmt/internal/TreePrinter.scala

```diff
@@ -503,7 +507,7 @@ object TreePrinter {
           case t: Pat.Extract =>
             dApplyParenPat(print(t.fun), t.args)
           case t: Pat.ExtractInfix =>
-            dInfix(mkPat(t.lhs), t.op.value, print(t.op), t.rhs.map(mkPat))
+            dInfixOld(mkPat(t.lhs), t.op.value, print(t.op), t.rhs.map(mkPat))
```

Comment:
Any reason we stick to old instead of new?

Reply:
I'm in the process of removing `dInfix`